index.html

```diff
@@ -32,14 +32,21 @@ <h5>Search History</h5>
           <div class="col l5" id="moviePoster"></div>
           <div class="col l7" id="movieInfo"></div>
         </div>
+        <div class="col m12" id="drinkHeading"></div>
         <div class="col m9" id="drinks"></div>
-            <!-- Lines 34-40 was trying carousel just appending images to see if it worked-it did! -->
             <div class="carousel">
               <a id="item1" class="carousel-item"></a>
               <a id="item2" class="carousel-item"></a>
               <a id="item3" class="carousel-item"></a>
               <a id="item4" class="carousel-item"></a>
               <a id="item5" class="carousel-item"></a>
+              <a id="item6" class="carousel-item"></a>
+              <a id="item7" class="carousel-item"></a>
+              <a id="item8" class="carousel-item"></a>
+              <a id="item9" class="carousel-item"></a>
+              <a id="item10" class="carousel-item"></a>
+              <a id="item11" class="carousel-item"></a>
+              <a id="item12" class="carousel-item"></a>
             </div>
```

Comment:
I think you already tried, but lines 38-49 look like they should be in the JavaScript file

Reply:
Yep I added .empty() for those as well. For loop is good for 12 items now.